File: lib/model/question.dart
class QuestionData {
  String? id;
  String? questionLink;
  String? title;
  int? answerTime;
  int? position;

  QuestionData(
    this.id,
    this.questionLink,
    this.title,
    this.answerTime,
    this.position,
  );

  Map toJson() => {
        'id': id,
        'question_link': questionLink,
        'title': title,
        'answer_time': answerTime,
        'position': position,
      };
}

File: android/src/main/kotlin/jp/oyster/camera_stream/models/Question.kt
package jp.oyster.camera_stream.models

import com.google.gson.annotations.SerializedName

data class Question(
    @SerializedName("id") val id: String,
    @SerializedName("title") val name: String?,
    @SerializedName("question_link") val audioLink: String?,
    @SerializedName("answer_time") val answerTime: Int? = 30,
    @SerializedName("position") val position: Int?
)
